kiwipy/rmq/threadcomms.py

```diff
@@ -179,6 +178,14 @@ def close(self):
         del self._loop
         self._closed = True
 
+    def add_connection_close_callback(self, callback: aio_pika.types.CloseCallbackType, weak: bool = False) -> None:
```

Comment:
Would you mind changing this to `add_close_callback` as this is used in the async version and by aio-pika so it'd be good to keep it consistent.

Reply:
My thinking here though was that `add_close_callback` gives the impression it is related to `RmqThreadCommunicator.close`, rather than `RmqThreadCommunicator._communicator.close`.
But if you still want I can change it?

Reply:
Yeah, I think so.  The `RmqThreadCommunicator._communicator` should be treated as an impementation detail and so it's OK if the user only 'sees' the closure of `RmqThreadCommunicator`

Reply:
no worries, done